redocly/scripts/combine-yamls.py

```diff
@@ -0,0 +1,136 @@
+import yaml
+import re
+import os
+
+# Paths to the API spec files to combine
+file_paths = {
```

Comment:
Nit: Name global constants in upper case to make it clear where constants are used in code